docs/SUMMARY.md

```diff
@@ -125,19 +117,25 @@
     * [Performance](guides/performance.md)
     * [Concurrency](guides/concurrency.md)
     * [Testing](guides/testing.md)
+    * [The shards command](the_shards_command/README.md)
     * [Writing Shards](guides/writing_shards.md)
         * [Hosting on GitHub](guides/hosting/github.md)
         * [Hosting on GitLab](guides/hosting/gitlab.md)
     * [Continuous Integration](guides/ci/README.md)
         * [GitHub Actions](guides/ci/gh-actions.md)
         * [CircleCI](guides/ci/circleci.md)
+    * [Using the compiler](using_the_compiler/README.md)
     * [Static Linking](guides/static_linking.md)
     * [Database](database/README.md)
         * [Connection](database/connection.md)
         * [Connection pool](database/connection_pool.md)
         * [Transactions](database/transactions.md)
+    * [Coding style](conventions/coding_style.md)
 * Tutorials
```

Comment:
Next we should add an overview page for *Tutorials*. This was previously skipped because there was only one. With classifying the getting started guide as a tutorial, we have two now.